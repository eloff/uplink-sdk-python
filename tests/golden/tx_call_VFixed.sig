r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 4601420616928295054870217369635752364447625019378212343215305854292753336969
