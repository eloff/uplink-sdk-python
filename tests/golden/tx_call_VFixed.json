{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": {
                            "contents": 3.223, 
                            "tag": "Fixed3"
                        }, 
                        "precision": 3, 
                        "tag": "VFixed"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AEw0NjAxNDIwNjE2OTI4Mjk1MDU0ODcwMjE3MzY5NjM1NzUyMzY0NDQ3NjI1MDE5Mzc4MjEyMzQzMjE1MzA1ODU0MjkyNzUzMzM2OTY5"
}